### PR TITLE
Fill some vending machines

### PR DESCRIPTION
**Pull request by Soundwavesghost**
Just filled up a couple vending machines before I had to go to work.

### CODE REVIEW

File: Resources/Prototypes/VendingMachines/chapel.yml [hunk 1]
```diff
@@ -1,5 +1,21 @@
-- type: vendingMachineInventory
+﻿- type: vendingMachineInventory
   id: PietyVend
   name: PietyVend
   description: "A vending machine containing religious supplies and clothing. A label reads: \"A holy vendor for a pious man.\""
   spriteName: chapel
+  startinginventory:
+    OuterclothingChaplainhoodie:1
```
Comment (ike709) [inline Resources/Prototypes/VendingMachines/chapel.yml]:
```suggestion
    OuterclothingChaplainhoodie: 1
```

Comment (Soundwavesghost) [reply Resources/Prototypes/VendingMachines/chapel.yml]: Changed made on my local copy.

**Comment by ike709** [comment]
#436

**Comment by Soundwavesghost** [comment]
Okay for most reactions that listed as needing Acetone...I just replaced that with Si (Silicon). Sugar = Guclose.

**Comment by PJB3005** [comment]
Could you split the chemicals and the vending machines into separate PRs, please?

**Comment by DamianX** [comment]
You committed a "default.xcf" file for some reason. I don't think it's supposed to be there.

**Pull request closed without merging**